from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_Lessons(object):
    def setupUi(self, Lessons):
        Lessons.setObjectName("Lessons")
        Lessons.resize(897, 679)
        self.centralwidget = QtWidgets.QWidget(parent=Lessons)
        self.centralwidget.setObjectName("centralwidget")
        self.lessons_label = QtWidgets.QLabel(parent=self.centralwidget)
        self.lessons_label.setGeometry(QtCore.QRect(0, 10, 901, 41))
        self.lessons_label.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.lessons_label.setObjectName("lessons_label")
        self.structure_label = QtWidgets.QLabel(parent=self.centralwidget)
        self.structure_label.setGeometry(QtCore.QRect(0, 59, 891, 31))
        self.structure_label.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.structure_label.setObjectName("structure_label")
        self.choose_label = QtWidgets.QLabel(parent=self.centralwidget)
        self.choose_label.setGeometry(QtCore.QRect(10, 120, 881, 31))
        self.choose_label.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.choose_label.setObjectName("choose_label")
        self.epochesbox = QtWidgets.QComboBox(parent=self.centralwidget)
        self.epochesbox.setGeometry(QtCore.QRect(250, 160, 400, 22))
        self.epochesbox.setObjectName("epochesbox")
        self.epochesbox.addItem("")
        self.epochesbox.addItem("")
        self.epochesbox.addItem("")
        self.epochesbox.addItem("")
        self.epochesbox.addItem("")
        self.epochesbox.addItem("")
        self.lessons_button = QtWidgets.QPushButton(parent=self.centralwidget)
        self.lessons_button.setGeometry(QtCore.QRect(300, 330, 281, 28))
        self.lessons_button.setObjectName("lessons_button")
        self.return_button = QtWidgets.QPushButton(parent=self.centralwidget)
        self.return_button.setGeometry(QtCore.QRect(350, 380, 181, 28))
        self.return_button.setObjectName("return_button")
        Lessons.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(parent=Lessons)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 897, 26))
        self.menubar.setObjectName("menubar")
        Lessons.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(parent=Lessons)
        self.statusbar.setObjectName("statusbar")
        Lessons.setStatusBar(self.statusbar)

        small_font = QtGui.QFont('Rockwell Extra Bold', 7)
        average_font = QtGui.QFont('Rockwell Extra Bold', 8)
        big_font = QtGui.QFont('Rockwell Extra Bold', 16)
        small_font.setBold(True)
        average_font.setBold(True)
        big_font.setBold(True)

        self.lessons_label.setFont(big_font)
        self.structure_label.setFont(small_font)
        self.choose_label.setFont(average_font)
        self.epochesbox.setFont(average_font)
        self.lessons_button.setFont(average_font)
        self.return_button.setFont(small_font)

        self.widgets = [self.lessons_label, self.structure_label,
                        self.choose_label, self.epochesbox, self.lessons_button,
                        self.return_button]

        self.retranslateUi(Lessons)
        QtCore.QMetaObject.connectSlotsByName(Lessons)

    def retranslateUi(self, Lessons):
        _translate = QtCore.QCoreApplication.translate
        Lessons.setWindowTitle(_translate("Lessons", "Lessons"))
        self.lessons_label.setText(_translate("Lessons", "Уроки"))
        self.structure_label.setText(_translate("Lessons", "Материал всех уроков структурирован по различным эпохам для упрощения понимания и изучения сложных исторических процессов"))
        self.choose_label.setText(_translate("Lessons", "Выберите эпоху:"))
        self.epochesbox.setItemText(0, _translate("Lessons", "Древний мир (до 500г н.э)"))
        self.epochesbox.setItemText(1, _translate("Lessons", "Средневековье (500-1500гг)"))
        self.epochesbox.setItemText(2, _translate("Lessons", "Эпоха Возрождения и Реформации (1500-1700гг)"))
        self.epochesbox.setItemText(3, _translate("Lessons", "Новое время (1700-1900гг)"))
        self.epochesbox.setItemText(4, _translate("Lessons", "XX век (1900-2000гг)"))
        self.epochesbox.setItemText(5, _translate("Lessons", "Современный мир (2000г - настоящее время)"))
        self.lessons_button.setText(_translate("Lessons", "Перейти к урокам по данной эпохе"))
        self.return_button.setText(_translate("Lessons", "Вернуться назад"))
